src/blight/cli.py

```diff
@@ -26,14 +29,32 @@ def _export_guess_wrapped():
         _export(f"BLIGHT_WRAPPED_{variable}", tool_path)
 
 
+def _swizzle_path():
+    blight_dir = Path(tempfile.mkdtemp(suffix="blight-XXX"))
+
+    for variable, tool in blight.tool.TOOL_ENV_MAP.items():
+        shim_path = blight_dir / tool
+        with open(shim_path, "w+") as io:
+            io.write(f'blight-{tool} "${{@}}"\n')
+
+        st = os.stat(shim_path)
```

Comment:
Let's do `shim_path.stat()` and `shim_path.chmod()` instead, since we're already using a `Path` object 🙂